### ISSUE TITLE
[FR] GPT soon ? 

### ISSUE THREAD

**Issue by ozanerturk** [mingo-app/mingo#633]
**Is your feature request related to a problem? Please describe.**
While knowing the schema with schema analyzer, It would be great to have an AI assistant building queries. I am doing it with external AI services and its frustrating to jump back and forth between mingo and browser.
 
**Describe the solution you'd like**
An AI chat panel may integrated in collection view 
**Describe alternatives you've considered**
- The AI assistant can generate queries, updates and performance/index recommendations  
- It can also generate widgets/dashboards etc.
- BI capability would be the best ex:
    - "Find unusual users activities" may produce queries for anomaly detection..
    - "Get most profitable products in last year"
    - ...

**Additional context**

**Comment by tothradoslav** [comment]
Yes, we are working on this. This will however effect the pricing options as LLMs are priced based on tokens.

**Comment by tothradoslav** [comment]
AI assistant has been implemented, but you must bring your own key.
